evergreen/prepare-ocsp.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with an error if any of the commands fail
+
+cd ${DRIVERS_TOOLS}/.evergreen/ocsp
+
+echo "Preparing Python env for OCSP tests"
+
+if [ "$OSTYPE" = "cygwin" ]; then
```

Comment:
because of this comment https://github.com/durran/mongo-csharp-driver/pull/2#discussion_r526987531, should we use `$OS`?

Reply:
Done